cluster/examples/kubernetes/ceph/cluster.yaml

```diff
@@ -60,11 +60,11 @@ spec:
     # v12 is luminous, v13 is mimic, and v14 is nautilus.
     # RECOMMENDATION: In production, use a specific version tag instead of the general v13 flag, which pulls the latest release and could result in different
     # versions running within the cluster. See tags available at https://hub.docker.com/r/ceph/ceph/tags/.
-    image: ceph/ceph:v13
+    image: ceph/daemon-base:latest-master
```

Comment:
This will be changed back, right?

Reply:
Okay, but before a merge and when the test is hopefully successful this will be changed to 12.2.10 image then?

Reply:
actually i'll just revert this entire cluster.yaml after i'm done testing. This setting will return to `ceph/ceph:v13` for master